### DISCUSSION TITLE
Some basic settings. Help

### DISCUSSION THREAD

**Discussion by t-e-s-tweb** [static-web-server/static-web-server#297] [Q&A] [answered]
Hello, I have a static site and i am trying to only use the simplest file server which in this case is sws. I need two main settings for now which are url redirect that is redirect from www.website.com to website.com. I have already found the settings for port 80 to 443.

**Comment by joseluisq** [comment]
Not possible with the current SWS but it'll be in the next release. See https://github.com/static-web-server/static-web-server/pull/301 so stay tuned.

**Reply by joseluisq** [reply, marked as answer]
Resolved by #301 and should be available in the next v2.25.0 release.
